lib/zstd_stats.h

```diff
@@ -0,0 +1,164 @@
+/*
+    zstd - standard compression library
```

Comment:
Yes, by default it shouldn't be included at all:

#if ZSTD_OPT_DEBUG == 3
    #include "zstd_stats.h"
#else